packages/material-ui/test/umd/run.js

```diff
@@ -4,46 +4,38 @@ const http = require('http');
 const path = require('path');
 const express = require('express');
 const expect = require('expect-puppeteer');
-const { addTeardown, shutdown } = require('../../../../modules/handleKillSignals');
```

Comment:
Are you should about the teardown logic? How do you make sure the browser is correctly close when sending an interruption signal do the program?

Reply:
How long does the script run for you?

Reply:
Also: you kind of can't. Orphaned processes are hard problem so you would need to show me a concrete case where this is a problem. A general solution is too much work for a little helper script.

Reply:
6,021 ms. Yeah ok, I see your point. For context, the teardown logic was previously used in Docker node.js files. We were using it for our web Next.js instances, our API express instances, and our end-to-end tests in puppeteer. We had different teardown to run in different orders, redis connection, postgres connection, node.js server handler, etc. I guess its overkill here.

Reply:
Node changed a lot regarding async/await logic. I'm fairly certain it takes care of possible orphan processes (and likely always did): You don't need to manually listen for sigint since any spawned processes are closed when the node process terminates.

Reply:
Agree with that. Our main use case was more for external collections and graceful shutdown (stop the http server first, then close the other connections).